### PR TITLE
Update old apt-get command

### PR DESCRIPTION
**Pull request by RukkiesMan**
### Proposed changes

This tutorial uses the old `apt-get` command. This commit updates that command to the new command `apt`.

### CODE REVIEW

File: install/linux/docker-ce/ubuntu.md [hunk 1]
```diff
@@ -204,41 +204,46 @@ from the repository.
...
     a. List the versions available in your repo:
 
     ```bash
-    $ apt-cache madison docker-ce
-
-      docker-ce | 5:18.09.1~3-0~ubuntu-xenial | {{ download-url-base }}  xenial/stable amd64 Packages
-      docker-ce | 5:18.09.0~3-0~ubuntu-xenial | {{ download-url-base }}  xenial/stable amd64 Packages
-      docker-ce | 18.06.1~ce~3-0~ubuntu       | {{ download-url-base }}  xenial/stable amd64 Packages
-      docker-ce | 18.06.0~ce~3-0~ubuntu       | {{ download-url-base }}  xenial/stable amd64 Packages
+    $ apt policy docker-ce
```
Comment (thaJeztah) [inline install/linux/docker-ce/ubuntu.md]: Wondering if we should use `apt list -a docker-ce` instead;

```bash
apt list -a docker-ce
Listing... Done
docker-ce/bionic,now 5:18.09.1~3-0~ubuntu-bionic amd64 [installed]
docker-ce/bionic 5:18.09.0~3-0~ubuntu-bionic amd64
docker-ce/bionic 18.06.1~ce~3-0~ubuntu amd64
docker-ce/bionic 18.06.0~ce~3-0~ubuntu amd64
docker-ce/bionic 18.05.0~ce~3-0~ubuntu amd64
docker-ce/bionic 18.03.1~ce~3-0~ubuntu amd64
```

**Comment by tianon** [comment]
Yeah I'm not sure on this, `apt` is officially discouraged for use in scripts and `apt-get` is likely to stay supported for a long time, so it might be worth mentioning both.

**Comment by bermudezmt** [comment]
Hello @tianon and @thaJeztah - let me know what you decide. My gut tells me we _*can*_ postpone the change, since it involves updating all Debian installation pages including EE, and making a note of the stability of `apt-get`. Happy to make the necessary changes though if we decide to proceed.

**Comment by thaJeztah** [comment]
Yes, perhaps we should postpone this change. I _think_ most users familiar with their Linux distro would know that `apt` and `apt-get` are interchangeable for these examples, so (if they prefer `apt`) would already use that.

For those that copy/paste the instructions from this page, it wouldn't make a big difference.

So, when in doubt, "no" is the safe answer (["no is temporary, yes is forever"](https://twitter.com/solomonstre/status/715277134978113536?lang=en))

**Comment by bermudezmt** [comment]
Thank you for linking that tweet @thaJeztah - such wisdom. 

@RukkiesMan Thank you for your feedback! We've decided not to incorporate your suggested changes at this time, but rest assured, we will consider this change on our content refactor.

**Pull request closed without merging**